packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -51,16 +51,29 @@ const getExistingLine = (
 
 export const recalculateSummary = (
   summaryItem: OutboundShipmentSummaryItem
-): { unitQuantity: number; numberOfPacks: number } => {
-  const unitQuantity = Object.values<OutboundShipmentRow>(
-    summaryItem.batches
-  ).reduce(getUnitQuantity, 0);
-
-  const numberOfPacks = Object.values<OutboundShipmentRow>(
-    summaryItem.batches
-  ).reduce(getSumOfKeyReducer('numberOfPacks'), 0);
+): Partial<OutboundShipmentSummaryItem> => {
```

Comment:
this was why the `[multiple]` wasn't working - the summary item was being created before all the batches were added and then not recalculated after they were. Moved that into the reducer and out of GeneralTab; likely will need to repeat this for inbound & reqs & stocktakes.